### PR TITLE
fix #80139 theblaze.com

### LINKED ISSUE #80139: www.theblaze.com
### Issue URL (Anti Adblock Script)
 [https://www.theblaze.com](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.theblaze.com)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Android 11
AdGuard product: | AdGuard for Android v3.6.1 (11)
Browser: | Brave
AdGuard mode: | VPN
Filtering quality: | High-quality
HTTPS filtering: | enabled
Stealth mode: | disabled
DNS filtering: | server: `Adguard`<br>
Filters: | <b>Ad Blocking:</b><br/> AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/> Fanboy's Enhanced Tracking List, <br/>AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/> EasyList Cookie List, <br/>Web Annoyances Ultralist<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist
Userscripts: | https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.meta.js
Other extensions used: | Brave ad blocker

### PR DESCRIPTION
**Pull request by wout-codes**
#80139

### CODE REVIEW

File: AnnoyancesFilter/sections/annoyances.txt [hunk 1]
```diff
@@ -1773,6 +1773,8 @@ revouninstaller.com,logincasino.com,borsagundem.com,officelife.media,kod.ru,mast
 ! ###mc_embed_signup
 kosicednes.sk,tinyprojects.dev,cambio16.com,voxukraine.org,pnas.org,cachevalleydaily.com,bisnis.com,ilpost.it,bsblog.info,alcoparty.com.ua,codastory.com,informazionefiscale.it,kerch.fm,cinemablend.com,prostofinovo.ru,comss.ru,majorgeeks.com,buhgalter.com.ua,providr.com,24tv.ua,babylonbee.com,dexerto.com,football365.com,menrec.com,netdoktor.de,ottawamatters.com,retail-insider.com,walterfootball.com,wweek.com,zozhnik.ru###mc_embed_signup
 !
+theblaze.com##.newsletter-2020
+theblaze.com##div[id*="Newsletter_Post_"]
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/annoyances.txt:1777]: What about this for all cases?
`theblaze.com##div[id*="_Newsletter_"]`

Comment (wout-codes) [reply AnnoyancesFilter/sections/annoyances.txt:1777]: It may be too wide and cause incorrect blocking.

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/annoyances.txt:1777]: I checked a lot of pages so I think we can change these 2 rules to 1

Comment (wout-codes) [reply AnnoyancesFilter/sections/annoyances.txt]: Repaired.

**Review by Sergey-Lyapin — APPROVED** [review]

**Pull request merged**